bigquery_etl/glean_usage/baseline_clients_daily.py

```diff
@@ -27,7 +27,7 @@ def run_query(
     project_id, baseline_table, date, dry_run, output_dir=None, output_only=False
 ):
     """Process a single table, potentially also writing out the generated queries."""
-    tables = table_names_from_baseline(baseline_table)
+    tables = table_names_from_baseline(baseline_table, include_project_id=False)
```

Comment:
Maybe we should have `table_names_from_baseline` include additional entries in the dictionary. Something like "daily_table_qualified" that would be the version with project ID. Then the view template could reference the fully qualified name while the query template could reference the "daily_table" unqualified name, etc.

Reply:
I might go with adding `project_id` as a variable that can be pulled in.